cachito/workers/tasks/gomod.py

```diff
@@ -16,45 +17,96 @@
 log = logging.getLogger(__name__)
 
 
+def _find_invalid_gomod_files(bundle_dir, subpaths):
```

Comment:
Would "missing" be more accurate than "invalid"?

Reply:
I rationalized that missing is invalid. But I guess the validity is handled outside this function (i.e. can we ignore missing). So it makes sense to change this.